src/components/Team/index.js

```diff
@@ -89,9 +103,14 @@ const Team = props => {
             <CircularProgress />
           </Box>
         )}
-      </Grid>
-    </Grid>
+      </Paper>
+      <Link to="/team/add">
+        <Fab color="primary" className={classes.fab}>
+          <AddIcon />
+        </Fab>
+      </Link>
+    </>
   );
 };
 
-export default Team;
+export default withStyles(styles)(Team);
```

Comment:
Is there any special reason to style the component using the HOC pattern instead of the hook pattern as we are using in other components?

Reply:
We should use the same. We have an issue now https://github.com/leusgalvan/fixture/issues/25